frontend/app/assets/svg.tsx

```diff
@@ -0,0 +1,376 @@
+export const SVGScriptCode = ({ className = '' }: { className?: string }) => (
+  <svg
+    className={`w-5 h-5 ${className}`}
```

Comment:
Should remove these default `w-N h-N` classes, and do so where components are added. That way, all `className` will stay at call-site, including color and size.